FrontEnd/src/tests/MainCompanies.test.js

```diff
@@ -0,0 +1,113 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import MainCompanies from '../components/landing-page/companies/Companies';
+
+afterEach(() => {
+  jest.resetAllMocks();
+});
+
+const results = [
+  {
+    id: 1,
+    name: 'saleonline',
+    categories: [
+      {
+        id: 1,
+        name: 'trade',
+      },
+      {
+        id: 2,
+        name: 'transport',
+      },
+    ],
+    region: 'Dnipro',
+    founded: 1980,
+    service_info: null,
+    address: 'Kyiv',
+    banner_image: null,
+    is_saved: true,
+  },
+  {
+    id: 2,
+    name: 'sale',
+    categories: [
+      {
+        id: 1,
+        name: 'trade',
+      },
+    ],
+    region: 'Dnipro',
+    founded: 2007,
+    service_info: null,
+    address: 'Dnipro',
+    banner_image: null,
+    is_saved: false,
+  },
+  {
+    id: 3,
+    name: 'PizzaHousesale',
+    categories: [],
+    region: 'Charkiv',
+    founded: null,
+    service_info: null,
+    address: 'Zaporija',
+    banner_image: null,
+  },
+  {
+    id: 4,
+    name: 'salefruits',
+    categories: [],
+    region: null,
+    founded: null,
+    service_info: null,
+    address: null,
+    banner_image: null,
+  },
+  {
+    id: 5,
+    name: 'salevegetables',
+    categories: [],
+    region: null,
+    founded: null,
+    service_info: null,
+    address: null,
+    banner_image: null,
+  },
+  {
+    id: 6,
+    name: 'salebushes',
+    categories: [],
+    region: null,
+    founded: null,
+    service_info: null,
+    address: null,
+    banner_image: null,
+  },
+];
+
+describe('MainCompanies component unit tests', () => {
+  test('renders new members results', () => {
+    jest.mock('axios');
+    const axios = require('axios');
```

Comment:
Could we make it in `beforeEach` method?

Reply:
Thank you for feedback. Done